sdk/communication/communication-sms/review/communication-sms.api.md

```diff
@@ -10,13 +10,67 @@ import type { KeyCredential } from '@azure/core-auth';
 import type { OperationOptions } from '@azure/core-client';
 import type { TokenCredential } from '@azure/core-auth';
 
+// @public (undocumented)
+export interface AddOptions extends OperationOptions {
+}
+
+// @public (undocumented)
+export interface CheckOptions extends OperationOptions {
+}
+
+// @public (undocumented)
+export interface OptOutAddResult {
+    // (undocumented)
+    errorMessage?: string;
+    // (undocumented)
+    httpStatusCode: number;
+    // (undocumented)
+    to: string;
+}
+
+// @public (undocumented)
+export interface OptOutCheckResult {
+    // (undocumented)
+    errorMessage?: string;
+    // (undocumented)
+    httpStatusCode: number;
+    // (undocumented)
+    isOptedOut: boolean;
+    // (undocumented)
+    to: string;
+}
+
+// @public (undocumented)
+export interface OptOutRemoveResult {
+    // (undocumented)
+    errorMessage?: string;
+    // (undocumented)
+    httpStatusCode: number;
+    // (undocumented)
+    to: string;
+}
+
+// @public (undocumented)
+export class OptOutsClient {
+    // Warning: (ae-forgotten-export) The symbol "SmsApiClient" needs to be exported by the entry point index.d.ts
+    constructor(api: SmsApiClient);
```

Comment:
Could you also export SmsApiClient? We shouldn't have such warnings.

Reply:
It is already exported: 
![image](https://github.com/user-attachments/assets/ddd22241-dfd4-40e0-a838-239461a45edf)

Reply:
You'll need to export it from the index.ts. You can do so by adding `export { SmsApiClient };` to the smsClient.ts file. However, that'll require exporting all the parts of the public API of `SmsApiClient` itself.

Reply:
Do you need a class for `OptOutsClient`?  How would a customer obtain a `SmsApiClient` instance in order to construct an `OptOutsClient`?  if the only way to have it is via `SmsClient.optOuts`, you may just export an interface for `OptOutsClient`

Reply:
@jeremymeng thanks for the reply, but could you please elaborate more? How do I just export an interface for OptOutsClient?

Reply:
something like below in smsClient.ts

```ts
import { OptOutsClient as OptOutsClientImpl } from "./optOutsClient.js";

export interface OptOutsClient {
  add(from: string, to: string[], options?: AddOptions): Promise<OptOutAddResult[]>;
  check(from: string, to: string[], options?: CheckOptions): Promise<OptOutCheckResult[]>;
  remove(from: string, to: string[], options?: RemoveOptions): Promise<OptOutRemoveResult[]>;
}

// ...

this.optOuts = new OptOutsClientImpl(this.api);

```

Reply:
Thank you, fixed